web-ui/src/main/resources/catalog/components/metadataactions/MetadataActionService.js

```diff
@@ -377,5 +377,28 @@
             md.getUuid();
         gnUtilityService.getPermalink(md.title || md.defaultTitle, url);
       };
+
+      /**
+       * Index the current selection of metadata records.
+       * @param {String} bucket
+       */
+      this.indexSelection = function(bucket) {
+        return $http.get('../api/records/index', {
+          params: {
+            bucket: bucket
+          }
+        }).then(function(response) {
+          var res = response.data;
+          gnAlertService.addAlert({
```

Comment:
This does not work on the editor board page because we still have 2 notifications systems. At some point, we'll have to stick to one only. I'll add the gnAlert to all apps for the time being.